cmake/Modules/DownloadProject.cmake

```diff
@@ -0,0 +1,174 @@
+# ~~~
+# Distributed under the OSI-approved MIT License.  See accompanying
```

Comment:
For all non-BSD licenses we have to update debian/copyright in the Debian branch.

Please do not create a cmake/Modules/LICENSE, it suggests that all cmake modules are MIT licensed, which they are not.